src/containers/Transaction/Transaction.js

```diff
@@ -0,0 +1,143 @@
+import React, { useState, useEffect } from 'react';
+import Switch from '@material-ui/core/Switch';
+import TextField from '@material-ui/core/TextField';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import Button from '@material-ui/core/Button';
+import { wrapper, heading, form, buttonContainer } from './transaction.module.scss';
```

Comment:
Few more classes and we will have a lot of imports here. All of them are available as a default also. So we may replace that with:
`import styles from 'transaction.module.scss'`